src/mito2/src/engine/flush_test.rs

```diff
@@ -221,3 +221,38 @@ async fn test_flush_empty() {
 ++";
     assert_eq!(expected, batches.pretty_print().unwrap());
 }
+
+#[tokio::test]
+async fn test_flush_reopen_region() {
```

Comment:
Can we add a new test that alters region and then flushes it? And reopen the region to check if it's ok.

Reply:
We already have this test case. I will add the sequence check to it.